edit-title/index.md

```diff
@@ -0,0 +1,14 @@
+## How do I edit my game title or description?
+
+If you want to make any changes to anything you've already entered, such as the game's title or description, do the following.
+
+1. Go to your [dashboard](http://gamejolt.com/dashboard/) and select the game from the list under "Your Games". You'll be whisked away to the game overview. 
+2. Click the icon (the 3 bars) next to "Overview" to see a menu of your options. 
```

Comment:
I think this changed, I'm not seeing the 3 bar menu. We should update these steps.